<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>331</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>229</green>
        <blue>229</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>229</green>
        <blue>229</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="title">
   <string/>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <property name="checkable">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="captionLabel">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Caption</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="imageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="waitUntilCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Отложить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="maximumDate">
        <date>
         <year>2015</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2014</year>
         <month>2</month>
         <day>18</day>
        </date>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::DaySection</enum>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="useAlbumRB">
       <property name="text">
        <string>Через альбом</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="directWallRB">
       <property name="text">
        <string>Сразу на стену</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" rowspan="3">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="ru9gagCheck">
          <property name="text">
           <string>ru9gag</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="geekCheck">
          <property name="text">
           <string>9geek</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="gifsCheck">
          <property name="text">
           <string>9gifs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="cuteCheck">
          <property name="text">
           <string>9cute</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="nsfwCheck">
          <property name="text">
           <string>9nsfw</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
